assets/sass/vendor/_mdc-switch.scss

```diff
@@ -29,4 +29,10 @@
 	+ label {
 		margin-left: 10px;
 	}
+
+	// Prevent the background checkbox from becoming visible when a switch
+	// loads or is set in the disabled state.
+	.mdc-switch__native-control::before {
```

Comment:
It looks like we already have some styles in place here for "Overriding WordPress load-styles.php" above on the native control. Can we merge this into that selector above? e.g. `&::before`